actionpack/lib/action_controller/metal/rate_limiting.rb

```diff
@@ -52,20 +86,34 @@ module ClassMethods
       #       rate_limit to: 3, within: 2.seconds, name: "short-term"
       #       rate_limit to: 10, within: 5.minutes, name: "long-term"
       #     end
-      def rate_limit(to:, within:, by: -> { request.remote_ip }, with: -> { head :too_many_requests }, store: cache_store, name: nil, **options)
+      def rate_limit(to:, within:, by: :rate_limit_by, with: :rate_limit_with, store: cache_store, name: nil, **options)
         before_action -> { rate_limiting(to: to, within: within, by: by, with: with, store: store, name: name) }, **options
       end
     end
 
     private
       def rate_limiting(to:, within:, by:, with:, store:, name:)
```

Comment:
I know it's silly but I keep wanting to have support for this:

```ruby
rate_limit to: 10.times, …
```

```suggestion
      def rate_limiting(to:, within:, by:, with:, store:, name:)
        to = to.size if to.is_a?(Enumerator) # Support 10.times
```

Reply:
I like that!